test/test-parse-statement.cpp

```diff
@@ -459,7 +459,7 @@ TEST(test_parse, if_without_body) {
     EXPECT_THAT(v.errors,
                 ElementsAre(ERROR_TYPE_FIELD(
                     error_missing_body_for_if_statement, if_and_condition,
-                    offsets_matcher(&code, 0, u8"if (a)"))));
+                    offsets_matcher(&code, 6, 7))));
```

Comment:
Almost-blocking: Use `strlen` and a string to define offset ranges, improving readability:
```suggestion
                    offsets_matcher(&code, strlen(u8"if (a)"), u8"\n"))));
```